components/layout_2020/geom.rs

```diff
@@ -26,7 +26,7 @@ pub type LengthOrAuto = AutoOr<Length>;
 pub type AuOrAuto = AutoOr<Au>;
 pub type LengthPercentageOrAuto<'a> = AutoOr<&'a LengthPercentage>;
 
-#[derive(Clone, Serialize)]
+#[derive(Clone, Copy, Serialize)]
```

Comment:
Then there is a bunch of existing `.clone()` that become unnecessary. Should they be removed here, or be left for a follow-up?

Reply:
Good point. In addition to the cleanup that we can do now that `AddAssign` and `SubAssign` don't need to take references for `LogicalVec`, I think this can be in a followup.